okta/resource_users.go

```diff
@@ -15,7 +15,38 @@ func resourceUsers() *schema.Resource {
 		Update: resourceUserUpdate,
 		Delete: resourceUserDelete,
 
+		CustomizeDiff: func(d *schema.ResourceDiff, v interface{}) error {
+			//for an existing user, the login field cannot change
+			prev, _ := d.GetChange("login")
+			if prev.(string) != "" {
+				if d.HasChange("login") {
```

Comment:
just wondering - does golang support compound boolean expressions in `if`? if so could you combined the `prev.(string) != ""` and `d.HasChange("login")` expressions into one if statment?